package.json

```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && babel --source-maps -d lib/ src/",
```

Comment:
oh, could we also maneuver `mocha.opts` into such a module?
Agian not related to this PR... just the nits...